src/flake8/main/options.py

```diff
@@ -104,7 +107,9 @@ def __str__(self) -> str:
         return "auto" if self.is_auto else str(self.n_jobs)
 
 
-def register_default_options(option_manager: OptionManager) -> None:
+def register_default_options(
+    option_manager: OptionManager, formatter_names: List[str]
```

Comment:
adding an argument here makes this harder to untangle later -- it also doesn't feel like the right place to do this

Reply:
I've refactored out it a bit in 2d456db938bea934a148f1433b9ca53876d85784. What do you think?